Scripts/Runtime/Entities/TaskDriver/TaskSet/TaskData/DataStream/Cancellation/CancelRequestsDataStream.cs

```diff
@@ -11,17 +11,22 @@ internal class CancelRequestsDataStream : AbstractDataStream,
         private readonly CancelRequestsDataSource m_DataSource;
         public ActiveLookupData<EntityProxyInstanceID> ActiveLookupData { get; }
 
-        public override uint ActiveID
+        public override DataTargetID DataTargetID
         {
-            get => ActiveLookupData.ID;
+            get => ActiveLookupData.WorldUniqueID;
+        }
+
+        public override IDataSource DataSource
+        {
+            get => m_DataSource;
         }
 
         public CancelRequestsDataStream(ITaskSetOwner taskSetOwner) : base(taskSetOwner)
         {
             TaskDriverManagementSystem taskDriverManagementSystem = taskSetOwner.World.GetOrCreateSystem<TaskDriverManagementSystem>();
             m_DataSource = taskDriverManagementSystem.GetCancelRequestsDataSource();
 
-            ActiveLookupData = m_DataSource.CreateActiveLookupData(TaskSetOwner);
+            ActiveLookupData = m_DataSource.CreateActiveLookupData(TaskSetOwner, "CANCEL_REQUEST");
```

Comment:
of all the times you don't choose to create a constant...
You'd avoid additional allocs per instance if you used a constant

Reply:
Yep, i forgot. Done.